cs/build/internal/Common.Internal.props

```diff
@@ -9,11 +9,13 @@
     <BOND_INCLUDE_PATH Condition="'$(BOND_INCLUDE_PATH)' == ''">$(MSBuildThisFileDirectory)..\..\..\idl</BOND_INCLUDE_PATH>
   </PropertyGroup>
   <Import Project="..\nuget\Common.props" />
-  <PropertyGroup Condition="'$(BuildFramework)' == ''">
+  <PropertyGroup Condition="'$(BuildFramework)' == '' and '$(UsingMicrosoftNETSdk)' != 'true'">
     <BuildFramework>net45</BuildFramework>
   </PropertyGroup>
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true'">
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+  </PropertyGroup>
+  <PropertyGroup>
     <OutputPath>bin\$(BuildType)\$(BuildFramework)</OutputPath>
     <IntermediateOutputPath>obj\$(BuildType)\$(BuildFramework)</IntermediateOutputPath>
     <FileAlignment>512</FileAlignment>
```

Comment:
@lalo, you said before that [not updating the MinimumVisualStudioVersion was OK](https://github.com/Microsoft/bond/pull/868#discussion_r186241552). Can we remove this property entierly?